grammar.js

```diff
@@ -2091,7 +2091,7 @@ module.exports = grammar({
 
       return seq(
         '.',
-        alias($._start_of_index_operator, '['),
+        field('method', alias($._start_of_index_operator, $.operator)),
         args,
         choice(']', ']?'),
```

Comment:
Should these be aliased to `$.operator` too?

Reply:
I don't think so, it seems surprising that a method would have more than one operator. I did just add a change to make sure these stay highlighted.